packages/showcases/css/stories/components/quantity/examples/overview.html

```diff
@@ -0,0 +1,116 @@
+<style>
+  .block {
+    margin: 20px;
+  }
+  body {
+    background-color: var(--vtmn-semantic-color_background-tertiary);
+  }
+</style>
+<div class="block">
+  <div class="vtmn-quantity">
+    <label for="my-quantity-1">Label</label>
+    <div class="vtmn-quantity_container">
+      <button
+        class="vtmn-btn vtmn-btn_variant--secondary vtmn-btn--icon-alone"
+        aria-label="subtract"
+      >
+        <span class="vtmx-subtract-line"></span>
+      </button>
+      <input type="number" id="my-quantity-1" />
+      <button
+        class="vtmn-btn vtmn-btn_variant--secondary vtmn-btn--icon-alone"
+        aria-label="add"
+      >
+        <span class="vtmx-add-line"></span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<div class="block">
+  <div class="vtmn-quantity">
+    <label for="my-quantity-2">Label</label>
+    <div class="vtmn-quantity_container">
+      <button
+        class="vtmn-btn vtmn-btn_variant--secondary vtmn-btn--icon-alone"
+        aria-label="subtract"
+        disabled
+      >
+        <span class="vtmx-subtract-line"></span>
+      </button>
+      <input type="number" id="my-quantity-2" />
+      <button
+        class="vtmn-btn vtmn-btn_variant--secondary vtmn-btn--icon-alone"
+        aria-label="add"
+      >
+        <span class="vtmx-add-line"></span>
+      </button>
+    </div>
+  </div>
+</div>
+
+<div class="block">
+  <div class="vtmn-quantity">
+    <label for="my-quantity-2">Label</label>
+    <div class="vtmn-quantity_container">
+      <button
+        class="vtmn-btn vtmn-btn_variant--secondary vtmn-btn--icon-alone"
+        aria-label="delete"
+      >
+        <span class="vtmx-delete-bin-line"></span>
```

Comment:
Why did you put a "delete bin"?
When I check the Figma, I only see "subtract" or "add".
![Capture d’écran 2021-12-31 à 01 54 27](https://user-images.githubusercontent.com/9600228/147796973-64241df2-0aef-4d0a-8240-7f26a2b5976a.png)

Reply:
Yes, it is a use case risen up by Ronan during the review. It is possible to change the icon in the button.

Reply:
In order to avoid confusion, I prefer to not add it in showcase. Let's let designers do this with illustrations in Zeroheight if they want.